src/config/dynamic-theme-fixes.config

```diff
@@ -585,6 +585,27 @@ body[style] {
 
 ================================
 
+bestbuy.ca
+
+INVERT
+.centerContainer
```

Comment:
Where can I find this element?

Reply:
https://www.bestbuy.ca/en-ca/category/gaming-monitors/474835